src/main/scala/sbtdatabricks/DatabricksHttp.scala

```diff
@@ -142,6 +143,147 @@ class DatabricksHttp(endpoint: String, client: HttpClient, outputStream: PrintSt
   }
 
   /**
+   * Create an execution context
+   * @param language the relevant coding language
+   * @param cluster the relevant cluster within which the context will be created
+   * @return The id of the execution context
+   *
+   */
+  private[sbtdatabricks] def createContext(language: DBCExecutionLanguage, cluster: Cluster): ContextId = {
+    outputStream.println(s"Creating '${language.is}' execution context on cluster '${cluster.name}'")
+    val post = new HttpPost(endpoint + CONTEXT_CREATE)
+    val form = new StringEntity(s"""{"language":"${language.is}","clusterId":"${cluster.id}"}""")
```

Comment:
I want to standardize these in case DB Api breaks compatibility. Could you please create case classes for these and use `mapper.writeValueAsString(...)`. I'm going to submit a PR for the others. You can maybe name them as `CreateContextRequestV1`, `DestroyContextRequestV1`, ...
and make them `private[sbtdatabricks]` under `sbtdatabricks.util.Requests`